float32 range   # range in meters
float32 bearing # bearing in radians
float32 roll    # roll difference, radians, should be 0
float32 pitch   # pitch difference, radians, should be 0
float32 yaw     # yaw difference, radians